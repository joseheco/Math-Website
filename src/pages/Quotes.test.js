import renderer from 'react-test-renderer';
import Quote from './Quotes';

describe('<Quote />', () => {
  it('it should match snapshot', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
