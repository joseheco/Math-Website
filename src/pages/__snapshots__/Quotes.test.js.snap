// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Quote /> it should match snapshot 1`] = `
<div
  className="quotes-div"
>
  <p
    className="quotes"
  >
    Loading..
    -
    Author
  </p>
</div>
`;
